drivers/digital-io/max22190/README.rst

```diff
@@ -0,0 +1,148 @@
+MAX22190 no-OS driver
```

Comment:
you should also source this file in sphinx/source similarly to what it is already done for other drivers